packages/core/src/components/Chips/Chips.tsx

```diff
@@ -48,8 +49,7 @@ export interface ChipsProps extends VibeComponentProps {
   iconClassName?: string;
   /** ClassName for left or right avatar */
   avatarClassName?: string;
-  // TODO Vibe 3.0: filter ElementAllowedColor.DARK_INDIGO, ElementAllowedColor.BLACKISH from colors which are valid for Chips
-  color?: ElementColor;
+  color?: Exclude<ElementAllowedColor, "dark_indigo" | "blackish">;
```

Comment:
We need to document it

Reply:
Today it is excluded from the storybook, do we still need to document it?

Reply:
I think we do, if it can be used anyhow it's a valid api

Reply:
Added